client/secureclient/src/test/java/com/linkedin/openhouse/client/ssl/TablesApiClientFactoryTest.java

```diff
@@ -63,20 +82,27 @@ public void testWebClientForHttpUrl() throws Exception {
 
   @Test
   public void testWebClientForHttpsUrl() throws Exception {
-    SslContext mockSslContext = Mockito.mock(SslContext.class);
-    File tmpCert = File.createTempFile("tmpcacert", ".crt");
-    tmpCert.deleteOnExit();
     ArgumentCaptor<String> truststoreLocationCapture = ArgumentCaptor.forClass(String.class);
-    TablesApiClientFactory tablesApiClientFactorySpy =
-        Mockito.spy(TablesApiClientFactory.getInstance());
-    Mockito.doReturn(mockSslContext).when(tablesApiClientFactorySpy).createSslContext(anyString());
 
     ApiClient apiClient =
         tablesApiClientFactorySpy.createApiClient(
             "https://test.openhouse.com", "", tmpCert.getAbsolutePath());
+
     Mockito.verify(tablesApiClientFactorySpy, Mockito.times(1))
         .createSslContext(truststoreLocationCapture.capture());
     assertNotNull(apiClient);
     assertEquals(tmpCert.getAbsolutePath(), truststoreLocationCapture.getValue());
   }
+
+  @Test
+  public void testSetClientNameCalled() throws Exception {
+    ArgumentCaptor<String> clientNameCapture = ArgumentCaptor.forClass(String.class);
+
+    tablesApiClientFactorySpy.setClientName("trino");
```

Comment:
small test to verify that when setClientName is called, apiClient is created, that the default headers of webclient (the core dependency of of apiclient) are set